drizzlepac/outputimage.py

```diff
@@ -491,25 +494,30 @@ def writeFITS(self, template, sciarr, whtarr, ctxarr=None,
                             hdu.header.append(_card)
                 hdu.header['filename'] = self.outweight
                 hdu.header['CCDCHIP'] = '-999'
+
                 if self.wcs:
-                    pre_wcs_kw = self.find_kwupdate_location(hdu.header,'CD1_1')
+                    pre_wcs_kw = self.find_kwupdate_location(hdu.header, 'CD1_1')
                     # Update WCS Keywords based on PyDrizzle product's value
                     # since 'drizzle' itself doesn't update that keyword.
-                    addWCSKeywords(self.wcs,hdu.header, blot=self.blot,
+                    addWCSKeywords(self.wcs, hdu.header, blot=self.blot,
                                    single=self.single, after=pre_wcs_kw)
 
+                # explicitly set EXTNAME, EXTVER to (sci,1)
+                hdu.header.set('EXTNAME', value='WHT', before='TELESCOP')
```

Comment:
Comment should be updated.

Reply:
Although this code has been replaced, the new keyword being added does include a comment as implemented in 89fa478.